server/rpcserver.go

```diff
@@ -35,9 +35,9 @@ func (r *rpcServer) RegisterWithGrpcServer(grpcServer *grpc.Server) error {
 	return nil
 }
 
-// QueryIsBlockBabylonFinalized is an RPC method that returns the finality status of a block by querying Babylon chain.
-func (r *rpcServer) QueryIsBlockBabylonFinalized(ctx context.Context, req *proto.QueryIsBlockBabylonFinalizedRequest) (*proto.QueryIsBlockFinalizedResponse, error) {
```

Comment:
do we have any external dependencies (e.g. op-node, finality-explorer, FP) that depends on this function?

Reply:
why are we not adding a new API but renaming the old one?

i see in `finalitygadget/interface.go` we added a new API. shouldn't we match it here?

Reply:
yes op e2e tests in fp repo call this function. the changes are:

- update `QueryIsBlockBabylonFinalized` to now call internal db
- move old logic to a new `QueryIsBlockBabylonFinalizedFromBabylon` function, which we can keep using for testing purposes, but wont be called by external services

Reply:
> update QueryIsBlockBabylonFinalized to now call internal db

wait I see `QueryIsBlockBabylonFinalized` is completely removed from `server/rpcserver.go`. Is that right?

shouldn't we have both `QueryIsBlockBabylonFinalizedFromBabylon` and `QueryIsBlockBabylonFinalized` in this file?